atlauncher/install

```diff
@@ -1,6 +1,4 @@
 #/usr/bin/bash
-if [ ! -f SetupATpi.sh ];then
 wget -nv https://raw.githubusercontent.com/pi-dev500/MinecraftMicrosoftPILauncher/main/setupATpi.sh || exit 1
-fi
-./SetupATpi.sh || exit 1
```

Comment:
why did you remove this line and the one bellow? now it only downloads the script and doesn't execute it if I understand it correctly.